module_overviews/be_mod1.md

```diff
@@ -17,14 +17,10 @@ Pairing, OOP, TDD, SRP, DRY code, basic data type creation/manipulation
 
 ## Projects
 
-<section>
-  <a href="https://backend.turing.io/module1/projects/" class="btn btn-dark">Back End M1 Projects</a>
-</section>
-
-Week 1: War or Peace OR Flashcards
-Week 2-3: Battleship
-Week 4-5: Futbol
-Week 6: Enigma OR Night Writer
+* [War or Peace](https://backend.turing.edu/module1/projects/war_or_peace)
+* [Mastermind](https://backend.turing.edu/module1/projects/mastermind)
+* [Black Thursday](https://backend.turing.edu/module1/projects/black_thursday)
+* [View All](https://backend.turing.io/module1/projects/)
```

Comment:
I updated the hyperlinks in my PR to match what the FE repo uses. The HTML I used is below and you can find the styles [here](https://github.com/turingschool/mentorship-curriculum-site/blob/757d37575942b0d45a300927a3fc55a665e11210/_sass/dtr.scss) (feel free to implement, however). We talked about it during the Committee Meeting and want to try and make them consistent, so let's update all of the **"Projects**" to use this styling.

```html
<ul>
  <li class="resource">
    <a href="/dtr/module-kick-off-dtr.html">Module Kick-off DTR</a>
  </li>
  <li class="resource">
    <a href="/dtr/module-exit-dtr.html">Module Exit DTR</a>
  </li>
</ul>
```

Reply:
Sounds good! Just updated!